extensions/amp-nexxtv-player/0.1/amp-nexxtv-player.js

```diff
@@ -217,7 +274,17 @@ class AmpNexxtvPlayer extends AMP.BaseElement {
       return;
     }
 
+    const eventType = data['event'];
+    if (!eventType) {
+      return;
+    }
+
+    if (eventType === 'playerready') {
+      this.playerReady_(data);
+    }
+
     redispatch(this.element, data['event'], {
```

Comment:
nit: reuse introduced helper

```suggestion
    redispatch(this.element, eventType, {
```

Reply:
done